src/main/java/com/team4/testingsystem/controllers/UsersController.java

```diff
@@ -31,9 +36,23 @@ public UsersController(UsersService usersService) {
                 .collect(Collectors.toList());
     }
 
+    @GetMapping("/employees")
+    public List<UserDTO> getAllUsers() {
+        return usersService.getAll().stream()
+                .map(UserDTO::new)
+                .peek(user -> user.setAssignedTest(findAssignedTest(user)))
```

Comment:
What do you think about one operation with getting all the assigned tests within one request and add them to user stream?